schema/spacewalk/upgrade/susemanager-schema-4.1.10-to-susemanager-schema-4.1.11/001-fix_suseChannelUserRoleView.sql

```diff
@@ -0,0 +1,82 @@
+-- Copyright (c) 2020 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+DROP VIEW IF EXISTS suseChannelUserRoleView;
```

Comment:
This comes from the original script, but I wonder... do we need it if we have the `CREATE OR REPLACE` below :-?